packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx

```diff
@@ -115,8 +116,14 @@ export const start = () => {
     return Promise.resolve()
         .then(() => new Promise((resolve) => loadableReady(resolve)))
         .then(() => {
-            ReactDOM.hydrate(<OuterApp {...props} />, rootEl, () => {
-                window.__HYDRATING__ = false
-            })
+            hydrateRoot(
+                rootEl,
+                <OuterApp
+                    {...props}
+                    callback={() => {
+                        window.__HYDRATING__ = false
```

Comment:
are we still using `window.__HYDRATING__`? if so, why do we need the `useIsHydrated` hook?